core/src/main/java/io/cucumber/core/plugin/NiceAppendable.java

```diff
@@ -11,9 +11,11 @@ final class NiceAppendable implements Appendable {
 
     private static final CharSequence NL = "\n";
     private final Appendable out;
+    private final boolean flushEveryWrite;
 
-    public NiceAppendable(Appendable out) {
+    public NiceAppendable(Appendable out, boolean flushEveryWrite) {
```

Comment:
There is only one user of the `NiceAppendable` that needs to flush. Consider using two constructors, one with 1-arg that sets `flushEveryWrite` to `false` and one 2-arg constructor.

Reply:
addressed in commit 4e020de.